src/devices/TitanmecLed/IDisplayOneByteCharacter.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Iot.Device.TitanmecLed
+{
+    /// <summary>
+    /// Contains methods to display character in one byte format. Usually used in 7 or 8 segment mode.
+    /// </summary>
+    public interface IDisplayOneByteCharacter
```

Comment:
do we need those interfaces? I'm a bit confused what's the purpose of this? How is that going to make user code simpler?

Reply:
For end users, no interface need to be worried IF THEY DON'T NEED THEM. In a simple single device environment, use the class (like TM1637) directly is enough. If multiple devices need to be supported, use interface will reduce the code duplication. Check sample2 for details.
For developers which are planned to expand more chips (still dozens to do) support from this company, interface with supported methods should be implemented to reduce the user side code.

Reply:
if user doesn't need the API it should not be public.

Reply:
> if user doesn't need the API it should not be public.

Check the sample2. That's the very common use in the user code side which need to support different kind of LED chips. Or, a program can only support one kind of chip. That's wired.

Reply:
Interfaces are removed. Now all methods are declared in base class.